upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -934,31 +927,16 @@ func (c *ApplyClusterCmd) validateKopsVersion() error {
 	}
 
 	if recommended != nil && !required {
-		fmt.Printf("\n")
-		fmt.Printf(starline)
-		fmt.Printf("\n")
-		fmt.Printf("A new kops version is available: %s\n", recommended)
-		fmt.Printf("\n")
-		fmt.Printf("Upgrading is recommended\n")
-		fmt.Printf("More information: %s\n", buildPermalink("upgrade_kops", recommended.String()))
-		fmt.Printf("\n")
-		fmt.Printf(starline)
-		fmt.Printf("\n")
+		glog.Infof("[UPDATE] A new kops version is available/recommended: %s\n", recommended)
```

Comment:
I like the [UPDATE] delineation, but I previously thought that the whitespace and starline were needed so that it really stood out to users.

Reply:
Happy to put this one back :-)

Reply:
Almost wondering if we should pull this out, and get it in another PR.  It is kinda outside the scope of the PR.